stat.go

```diff
@@ -168,10 +174,10 @@ func (fs FS) Stat() (Stat, error) {
 	fileName := fs.proc.Path("stat")
 	data, err := util.ReadFileNoStat(fileName)
 	if err != nil {
-		return Stat{}, err
+		return newStat(), err
 	}
 
-	stat := Stat{}
+	stat := newStat()
```

Comment:
I think you can just do an in-line initialization here instead of the call to newStat()